homeassistant/components/august/__init__.py

```diff
@@ -141,15 +144,28 @@ async def async_setup(self):
         self._pubnub_unsub = async_create_pubnub(user_data["UserID"], pubnub)
 
         if self._locks_by_id:
-            tasks = []
-            for lock_id in self._locks_by_id:
-                detail = self._device_detail_by_id[lock_id]
-                tasks.append(
+            # Do not prevent setup as the sync can timeout
+            # but it is not a fatal error as the lock
+            # will recover automatically when it comes back online.
+            asyncio.create_task(self._async_initial_sync())
+
+    async def _async_initial_sync(self):
+        """Attempt to request an initial sync."""
+        # We don't care if this fails because we only want to wake
+        # locks that are actually online anyways and they will be
+        # awake when they come back online
+        with contextlib.suppress(
+            (asyncio.TimeoutError, ClientResponseError, CannotConnect)
+        ):
+            await asyncio.gather(
```

Comment:
A `gather` will cancel other tasks when 1 task raises. From the description it sounds like you might want to ensure that the requests still go through ?

You can do this by passing `return_exceptions=True` and raise any returned exception that is not supposed to be suppresed.